### DISCUSSION TITLE
Jobs/runners get killed after 16 minutes

### DISCUSSION THREAD

**Discussion by sylvestre** [community/community#40200] [Actions]
### Select Topic Area

Bug

### Body

Recently, some long jobs (usually taking an hour) are being killed after 16 minutes.
The message is:

```
The runner has received a shutdown signal. This can happen when the runner service is stopped, or a manually started runner is canceled.
```

Example:
https://github.com/uutils/coreutils/actions/runs/3557848920

Is it a recent change/limitation ?

Thanks

**Comment by ossanna16** [comment]
Hi there @sylvestre and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Reply by vitalykarasik** [reply]
It seems to me like a GitHub outage, I saw that from time to time.

Check https://www.githubstatus.com/history

**Comment by AaronCrawfis** [comment]
My team is also hitting this issue on Ubuntu 20.04:
 
<img width="1092" alt="image" src="https://user-images.githubusercontent.com/54363786/205168275-d68d87f5-cd16-4a95-9beb-869021737af2.png">

I found some other links with similar issues:

- https://github.com/orgs/community/discussions/26830
- https://github.com/actions/runner/issues/724
- https://github.com/actions/runner/issues/2241
